src/components/common/vertical-toolbar.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import Toolbar from './toolbar';
+
+const VerticalToolbar = styled(Toolbar)`
+  flex-direction: column;
+  padding: 0 24px;
+  
+  .toolbar-item {
+    padding: 16px;
```

Comment:
vertical toolbar `toolbar-item` padding should be `16px 0`. Because toolbar already has side padding